### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by ian-antking**

### CODE REVIEW

File: __tests__/kata1.fizzBuzz.test.js [hunk 1]
```diff
@@ -2,18 +2,21 @@ const { fizzBuzz } = require('../src');
 
 describe('fizzBuzz', () => {
   it('returns Fizz when passed a multiple of 3', () => {
-
+      expect(fizzBuzz(3)).toBe('Fizz')
```
Comment (ian-antking) [inline __tests__/kata1.fizzBuzz.test.js:5]: It's worth adding a few more tests with different multiples of three.

Comment (ian-antking) [reply __tests__/kata1.fizzBuzz.test.js:5]: A few more multiples of 5 would be a good idea too.

File: __tests__/kata4.humanCatDogYears.test.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
 const { humanCatDogYears } = require('../src');
+describe('humanCatDogYears', () =>{ 
+    it('returns an array of human, cat & dog years')
+    expect(humanCatDogYears(10)).toEqual([10,56,64])
```
Comment (ian-antking) [inline __tests__/kata4.humanCatDogYears.test.js:4]: Try adding a few more tests. What happens if you only enter one or two years?

File: src/kata1.fizzBuzz.js [hunk 3]
```diff
@@ -1,5 +1,20 @@
 const fizzBuzz = (number) => {
+    const isDivisibleBy3 = ( number % 3) === 0
+    const isDivisibleBy5 = ( number % 5) === 0
```
Comment (ian-antking) [inline src/kata1.fizzBuzz.js:4]: Good thinking separating this logic out from the `if` statements!

File: src/kata3.numberToReversedDigits.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 const numberToReversedDigits = (number) => {
-
+   const numberArray = Array.from(number.toString()).map(Number)
```
Comment (ian-antking) [inline src/kata3.numberToReversedDigits.js:2]: `number.toString().split('')` would also work. For the sake of readability it might make sense to split this over a few lines.

File: src/kata4.humanCatDogYears.js [hunk 5]
```diff
@@ -1,5 +1,16 @@
...
+        return 24 + ((number - 2) * 4)
+    };
+
+    let dogYears = (number) => {
+        return 24 + ((number - 2) * 5 )
+    };
+
+    yearsArray.push(number, catYears(number), dogYears(number))
+    return yearsArray
+};
```
Comment (ian-antking) [inline src/kata4.humanCatDogYears.js:14]: I can see this working for any numbers other than 1 or 2.

File: src/kata6.joinNames.js [hunk 6]
```diff
@@ -1,5 +1,8 @@
-const joinNames = (namesObj) => {
-
+const joinNames = (objectArray) => {
+   let namesArray = objectArray.map(element => element.name)
+    let finalName = namesArray.pop().toString()
```
Comment (ian-antking) [inline src/kata6.joinNames.js:3]: I like the use of `pop()`, very simple.

File: __tests__/kata7.getEmployerRole.test.js [hunk 7]
```diff
@@ -1,7 +1,28 @@
...
 
 describe('getEmployerRole', () => {
+const employees = [{
+
+    name: 'Holly',
+  
+    role: 'Category Manager'
+  
+  }, {
+
```
Comment (ian-antking) [inline __tests__/kata7.getEmployerRole.test.js:11]: I don't think you need to leave empty spaces between lines here. This is just a style preference though.

**Review by ian-antking — COMMENTED** [review]
Looking good. Just a few missed opportunities to test some edge cases.  There are a few places where your code could be a bit more readable, but this will come in time.

Well done!

**Review by HollyRebekah — APPROVED** [review]
